forem/forem
# Replace the "setup onboarding profile fields" temp rake task with a data update script

## What type of PR is this? (check all applicable)


## Description
As per [this discussion](https://dev-internal.slack.com/archives/C0134R988P9/p1607420168217600) rather than updating the profile fields for onboarding (for existing forems) via a script that was created in [this PR](https://github.com/forem/forem/pull/11704), we want to swap it out for a data update script so that it can run automatically.

## Related Tickets & Documents
https://github.com/forem/forem/pull/11704
https://github.com/forem/forem/issues/11529

## QA Instructions, Screenshots, Recordings
I'm not sure what else can be done except to run the spec to make sure it works.

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?

      have not been included_

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
